tests/remote-files/RCX_01_shortened_v2.txt

```diff
@@ -0,0 +1,3 @@
+https://gitlab.ics.muni.cz/umsa/umsa-files/-/raw/aplcms_testdata/testdata/recetox-aplcms/RCX_01_shortened_v2/RCX_01_shortened_v2.mzML
+https://gitlab.ics.muni.cz/umsa/umsa-files/-/raw/aplcms_testdata/testdata/recetox-aplcms/RCX_01_shortened_v2/RCX_01_shortened_v2_cdf.Rds
+https://gitlab.ics.muni.cz/umsa/umsa-files/-/raw/aplcms_testdata/testdata/recetox-aplcms/RCX_01_shortened_v2/RCX_01_shortened_v2_features.Rds
```

Comment:
Actually I noticed the links do not work properly. I think it should be 

```https://gitlab.ics.muni.cz/umsa/umsa-files/-/raw/master/testdata/recetox-aplcms/RCX_01_shortened_v2/RCX_01_shortened_v2.mzML```

(or maybe even permalink instead of `master`).

Reply:
Ah yeah, the PR got merged - good point!